doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -109,9 +110,16 @@ The following best practices have been adopted to ensure verification and valida
 <a name="1.6"></a>
 ## 1.6 Verification methodologies
 
-Perform VNF interoperability verifications against an implementation of CNTT reference architecture, leveraging existing OPNFV Intake Process. Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
+Prior to onboarding a VNF the NFVi must be tested in Network, Compute and Storage domains to ensure that it provides required capabilities that allow the VNF to behave as expected.
```

Comment:
I believe this statement refers to having fully tested a CNTT RA-x, and passing all compliance and conformance tests before proceeding with VNF(CNF) on-boarding.  If true, then please consider refactoring portions of the original sentence to clarify.  

For example, "Prior to on-boarding a VNF(CNF) the NFVI must be tested in a compliant reference architecture consisting of Network, Compute and Storage domains, to ensure the NFVI provides required exposure of features and capabilities which meet VNF(CNF) developer functional and platform requirements."

Reply:
@mf4716 I'm not sure I quite follow the suggestion.  Is this suggesting that the NFVI design may be tested in a particular location, and the design itself receive the certification that would then apply to different NFVI deployments?

Reply:
@kedmison - test and receive certification badge for the NFVI; then, deploy, test, and certify the vnf on the same NFVI, receiving the vnf badget after.  The way the statement was modified, it's not clear NFVI badging would be a pre-requisite to VNF badging.

Reply:
@mf4716 @kedmison @acmacm @LucProvoost 

As updated 1.6 Verification methodologies ... only refers to NFVI verification. Statement (ln 113) describes an approach for verifying an NFVI to compare its capabilities against a CNTT RI (the reference). This is independent of an actual VNF that will ultimately be on-boarded (commercially deployed environment). Not to be confused with testing a CNTT RA-x (instantiate as an RI-x) which uses normal code testing/validation methods such as API conformance etc. On that point I think we agreed not refer to "conformance" testing in OVP since that implies testing against a specification or standard (e.g. conformance with a protocol). We are testing against a CNTT RI which instantiates a RA. Perhaps we should title it NFVI Verification methodology and make a separate heading for VNF verification methodology as the methodologies are distinct?

I don't think NFVI badging is a pre-requisite to VNF badging since the whole point is to decouple VNF from NFVI. I should be able to verify a VNF against the reference i.e. independent of any commercial NFVI.

Reply:
> We are testing against a CNTT RI which instantiates a RA. Perhaps we should title it NFVI Verification methodology and make a separate heading for VNF verification methodology as the methodologies are distinct?

Agree @trevgc .  The paragraphs removed referenced a holistic methodology.  Versus, the recommended replacement verbiage which pertains to VNF testing only.  Recommended adding back the holistic verbiage, and creating a sub-section here which contains the new VNF info.

> I don't think NFVI badging is a pre-requisite to VNF badging since the whole point is to decouple VNF from NFVI. I should be able to verify a VNF against the reference i.e. independent of any commercial NFVI.

VNF onboarding acceleration will occur when Telcos accept VNFs validated against a CNTT normalized architecture.  Two badges, NFVI and VNF, ultimately achieve CNTT's target-end state goal.   VNF badges should only be granted when executed against the discrete architecture(s) defined by CNTT.  Anything different continues to perpetuate platform fragmentation and interconnection challenges Telcos deal with today.

Reply:
@trevgc - can we revert/refactors some of the existing language so that we maintain the original intent of the chapter/paragraphs?